sherpa-onnx/csrc/online-recognizer.cc

```diff
@@ -210,9 +215,26 @@ class OnlineRecognizer::Impl {
                                             features_vec.size(), x_shape.data(),
                                             x_shape.size());
 
+    std::array<int64_t, 1> processed_frames_shape{(int64_t) all_processed_frames.size()};
```

Comment:
`all_processed_frames.size()` returns an unsigned integer. Static cast is required to avoid compiler warnings. I changed the code to look like

```cpp
    std::array<int64_t, 1> processed_frames_shape{
      static_cast<int64_t>(all_processed_frames.size())};
```

Let me know if that looks good. Thanks.